tests/test_scope_serialization.py

```diff
@@ -18,24 +19,32 @@ class TestScopeSerialization:
     )
     admin_route_objects = (
         [
-            {"name": "CR", "additionalInformation": "knora-admin:SystemAdmin", "permissionCode": None},
-            {"name": "V", "additionalInformation": "knora-admin:CustomGroup", "permissionCode": None},
+            {"name": "CR", "additionalInformation": f"{KNORA_ADMIN_ONTO_NAMESPACE}SystemAdmin", "permissionCode": None},
```

Comment:
Why do you add the namespace only to replace it later? you could also create `KNORA_ADMIN_PREFIX = "knora-admin:"`

Reply:
I'm not aware that I replace it later, at least not here in this file.

But more generally, the problem is that we use the same `class Group(BaseModel)` for all operations on all routes. In https://github.com/dasch-swiss/dsp-permissions-scripts/pull/113, I changed this class, so that it stores the groups in the relative form `knora-admin:xyz`. This works well for most API routes, but not for the `/admin/permissions` route. Only for this specific route, we have to use absolute IRIs.